assets/app/scripts/directives/events.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('events', function($routeParams, $filter, DataService, ProjectsService, Logger) {
+    return {
+      restrict: 'E',
+      scope: {
+        resourceKind: "@",
+        resourceName: "@",
+        parentScope: "="
+      },
+      templateUrl: 'views/directives/events.html',
+      controller: function($scope){
+
+        var filterEvent = function(event) {
+          var resourceName = $scope.resourceName;
+          var resourceKind = $scope.resourceKind;
+          // For Build resource watch events from the builder Pod
+          if (resourceKind === "Build") {
+            resourceKind = "Pod";
+            resourceName = $scope.parentScope.build.metadata.annotations["openshift.io/build.pod-name"];
+          } else if (resourceKind === "Deployment") {
+            // For Deployment resource watch ReplicationController events
+            resourceKind = "ReplicationController";
```

Comment:
We should just pass in "ReplicationController" to the directive. "Deployment" is not really a kind.